tools/metadata-colon-converter.js

```diff
@@ -0,0 +1,88 @@
+var _ = require('underscore');
+var files = require('./files.js');
+
+// scheme - Object, representing paths to correct. Ex.:
+// {
+//   format: false,
+//   arch: false,
+//   load: [
+//     {
+//       node_modulus: true,
+//       sourceMap: true,
+//       sourceMapRoot: true,
+//       path: true
+//     }
+//   ]
+// }
+var convertByScheme = function (val, scheme) {
+  if (scheme === true)
+    return convert(val);
+  else if (scheme === false)
+    return val;
+
+  if (_.isArray(scheme)) {
+    if (! _.isArray(val))
+      throw new Error("Expected an array");
+
+    return _.map(val, function (subval, i) {
+      return convertByScheme(subval, scheme[0]);
+    });
+  }
+
+  if (! _.isObject(scheme))
+    throw new Error("Unexpected type of scheme: " + typeof(scheme));
+
+  var ret = _.clone(val);
+  _.each(scheme, function (subscheme, key) {
+    if (_.has(ret, key))
+      ret[key] = convertByScheme(val[key], subscheme);
+  });
+
+  return ret;
+};
+
+var convert = function (str) {
+  return files.adaptLegacyPath(str);
+};
+
+var ISOPACK_SCHEME = {
```

Comment:
I think the converter checks that input matches the scheme and throws an exception if it doesn't. So if the format changes, people would immediately get an error on the first test run.